src/components/carousel.js

```diff
@@ -18,7 +19,9 @@ export default class CarouselClass extends Component {
             pictureTops: [],
             pictureBottoms: [],
             currentTopIndex: 0,
-            currentBottomIndex: 0
+            currentBottomIndex: 0,
+            idTop: 0,
```

Comment:
This should probably be something like:
`idTop: getIdFromIndex(currentTopIndex, picture Tops)`
Then you'll define this function to get the clothing item's id, handling the edge case of an empty array. E.g.:
```
const getIdFromIndex = (index, items) => {
  if (index.length === 0) return null;
  return items[index].id;
}
```